modules/core/src/main/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStore.java

```diff
@@ -895,7 +895,7 @@ private void wakeUp() {
         protected Thread thread;
 
         /** Cache flushing frequence in nanos. */
-        protected long cacheFlushFreqNanos = cacheFlushFreq * 1000;
+        protected long cacheFlushFreqNanos = cacheFlushFreq * 1000 * 1000;
```

Comment:
Just wondering, do we have some constant like NANOS_IN_MILLIS in Ignite?

Reply:
No such public constants are present in Ignite, we have private NANOS_IN_MICROSECOND in CacheMetricsImpl and NANOS_IN_SECOND in IntervalBasedMeasurement.

Reply:
May be in future we should create 1 place for such constants. Thank you for reply